js/src/pages/settings/enhanced-conversions/setup-enhanced-conversions.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import { CheckboxControl, Notice } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppDocumentationLink from '~/components/app-documentation-link';
+import { useEnableEnhancedConversions } from './useEnableEnhancedConversions';
+
+/**
+ * Internal dependencies
+ */
+import Section from '~/components/section';
+
+const SetupEnhancedConversions = () => {
+	const [ isEnabled, , toggleEnhancedConversions ] =
+		useEnableEnhancedConversions();
+
+	const [ isNoticeVisible, setIsNoticeVisible ] = useState( false );
+	const [ isSaving, setIsSaving ] = useState( false );
+
+	const onChangeECState = async () => {
+		setIsSaving( true );
+		await toggleEnhancedConversions();
+		setIsNoticeVisible( true );
+		setIsSaving( false );
+	};
+
+	return (
+		<Section
+			title={ __(
+				'Improve conversion accuracy',
+				'google-listings-and-ads'
+			) }
+			description={
+				<div>
+					<p>
+						{ __(
+							'Enhanced Conversions is a feature designed to improve your measurement accuracy by collecting privacy-conscious data without the need for third-party cookies.',
+							'google-listings-and-ads'
+						) }
+					</p>
+					<p>
+						<AppDocumentationLink
+							href="https://support.google.com/google-ads/answer/9888656"
+							context="setup-enhanced-conversions"
+							linkId="enhanced-conversions-read-more"
+						>
+							{ __( 'Read more', 'google-listings-and-ads' ) }
+						</AppDocumentationLink>
+					</p>
+				</div>
+			}
+		>
+			<Section.Card className="gla-settings-enhanced-conversions">
```

Comment:
@ankitrox Do we need the `gla-settings-enhanced-conversions` class name? I don't see any styles for that class name.

Reply:
I don't think it is required. Removed.